openapi.yaml

```
@@ -280,4 +294,116 @@ components:
         type: string
         example: Team A
       example: ["All teams", "Team A", "Team B"]
-
+    EnvironmentsGetResponse:
+      type: object
+      properties:
+        id:
+          title: id
+          description: Environment identifier
+          type: string
+          example: "1"
+        name:
+          title: Name
+          description: Environment name
+          type: string
+          example: DEV
+        type:
+          title: Type
+          description: Environment type
+          type: string
+          enum: [kafka, kafkaconnect, schema]
+        tenantId:
+          title: TenantId
+          description: Tenant identifier
+          type: number
+          example: 101
+        topicprefix:
+          title: Topic prefix
+          description: Topic name prefix
+          type: string
+          nullable: true
+          example: "test-"
+        topicsuffix:
+          title: Topic suffix
+          description: Topic name suffix
+          type: string
+          nullable: true
+          example: "-test"
+        clusterId:
+          title: Cluster identifier
+          type: number
+          example: 1
+        tenantName:
+          title: Tenant name
+          type: string
+          example: default
```

```suggestion
          example: "default"
```

Maybe this can also be a string?